telegram/payment/invoice.py

```diff
@@ -22,17 +22,27 @@
 
 
 class Invoice(TelegramObject):
-    """This object contains basic information about an invoice.
+    """
+    This object contains basic information about an invoice.
 
     Attributes:
-        title (str): Product name
-        description (str): Product description
-        start_parameter (str): Unique bot deep-linking parameter that can
-            be used to generate this invoice
-        currency (str): Three-letter ISO 4217 currency code
-        total_amount (int): Total price in the smallest units of the currency (integer)
-        **kwargs (dict): Arbitrary keyword arguments.
-
+        title (:obj:`str`): Product name.
+        description (:obj:`str`): Product description.
+        start_parameter (:obj:`str`): Unique bot deep-linking parameter.
+        currency (:obj:`str`): Three-letter ISO 4217 currency code.
+        total_amount (:obj:`int`): Total price in the smallest units of the currency.
+
+    Args:
+        title (:obj:`str`): Product name.
+        description (:obj:`str`): Product description.
+        start_parameter (:obj:`str`): Unique bot deep-linking parameter that can be used to
+            generate this invoice.
+        currency (:obj:`str`): Three-letter ISO 4217 currency code.
+        total_amount (:obj:`int`): Total price in the smallest units of the currency (integer, not
+            float/double). For example, for a price of US$ 1.45 pass amount = 145. See the exp
+            parameter in currencies.json, it shows the number of digits past the decimal point for
```

Comment:
Please link currencies.json if you reference it
This comment also applies elsewhere.